From cba04d8568eb2f5b1e88d31e34b0ac3d0ce33c8f Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 12 Sep 2012 13:16:40 +0000
Subject: [PATCH 0347/1077] use rules from
 SelfServiceBuild\Build\cpp\makefile.rules.mak to compile generated sources

git-svn-id: svn://localhost/SelfServiceCommon/trunk@413 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/LogService/makefile.mak            | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/LogService/makefile.mak b/SelfServiceCommon/Massai/cpp/LogService/makefile.mak
index cd1244c0..c8f1e812 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/LogService/makefile.mak
@@ -313,13 +313,14 @@ MY_BIN8_RES = \
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
 
+$(_OBJ)\eventConfiguration.obj: $(_LOCAL_GEN)\eventConfiguration.cpp
 
 .SUFFIXES : .txt
 
-# txt => cpp => obj
-{.}.txt{$(_OBJ)}.obj:
-    echo #include "eventConfiguration.h" > $(_OBJ)\$(@B).cpp 
-    echo void initializeEventConfiguration(EventDefinition^& def) { >> $(_OBJ)\$(@B).cpp 
-    sed -f $(_SRC)\convert.sed $< >> $(_OBJ)\$(@B).cpp 
-    echo } >> $(_OBJ)\$(@B).cpp 
-    $(CC) $(STDCFLAGS) $(MY_CFLAGS) $(_OBJ)\$(@B).cpp -Fo$@
+# txt => cpp
+{.}.txt{$(_LOCAL_GEN)}.cpp:
+    -mkdir $(_LOCAL_GEN)
+    echo #include "eventConfiguration.h" > $(_LOCAL_GEN)\$(@B).cpp 
+    echo void initializeEventConfiguration(EventDefinition^& def) { >> $(_LOCAL_GEN)\$(@B).cpp 
+    sed -f $(_SRC)\convert.sed $< >> $(_LOCAL_GEN)\$(@B).cpp 
+    echo } >> $(_LOCAL_GEN)\$(@B).cpp
-- 
2.41.0.windows.1

